diff --git a/node_modules/named-placeholders/index.js b/node_modules/named-placeholders/index.js
index 3524ef5..6b64ad8 100644
--- a/node_modules/named-placeholders/index.js
+++ b/node_modules/named-placeholders/index.js
@@ -40,7 +40,7 @@ function parse(query) {
               continue;
             }
             inQuote = false;
-          } else if (chr === DQUOTE || chr === SQUOTE) {
+          } else if (!inQuote && (chr === DQUOTE || chr === SQUOTE)) {
             inQuote = true;
             qchr = chr;
           }
